gen/gen.go

```diff
@@ -449,9 +449,9 @@ func (g *Gen) writeGoDoc(packageName string, output io.Writer, swagger *spec.Swa
 	return err
 }
 
-var packageTemplate = `// Package {{.PackageName}} GENERATED BY SWAG; DO NOT EDIT
-// This file was generated by swaggo/swag{{ if .GeneratedTime }} at
-// {{ .Timestamp }}{{ end }}
+var packageTemplate = `// Code generated by SWAG. DO NOT EDIT
+// Package {{.PackageName}}
+// {{ if .GeneratedTime }} at {{ .Timestamp }}{{ end }}
```

Comment:
The phrase containing the generator package name and time stamp makes more sense to me.
```go
var packageTemplate = `// Code generated by swaggo/swag{{ if .GeneratedTime }} at {{ .Timestamp }}{{ end }}. DO NOT EDIT
// Package {{.PackageName}}`
```

Reply:
I changed it, can you check?